molsim/utils.py

```diff
@@ -263,7 +271,7 @@ def find_peaks(freq_arr,int_arr,res,min_sep,is_sim=False,sigma=3,kms=True):
 	indices = signal.find_peaks(int_new,distance=chan_sep)
 
 	if kms is True:
-		indices = [find_nearest(freq_arr,freq_new[x]) for x in indices[0]] #if we had to re-sample things
+		indices = find_nearest_vectorized(freq_arr,freq_new[indices[0]]) #if we had to re-sample things
```

Comment:
Can you confirm this has the same behavior? This doesn't look equivalent to me...

Reply:
Hmm, you're right, I mistakenly thought that my code called the function so when it ran properly I thought it was fine. Pushed an update that reverted it.